android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -233,6 +234,7 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
         } else {
             holder.sessionBookmarkIcon.setImageResource(R.drawable.ic_bookmark_border_white_24dp);
         }
+        DrawableCompat.setTint(holder.sessionBookmarkIcon.getDrawable(), ContextCompat.getColor(context, Integer.valueOf(session.getTrack().getFontColor())));
```

Comment:
@iamareebjamal 
but how to i access the R.color when the value is to be taken from json file?